### DISCUSSION TITLE
Vercel App Appears Blank On iPhone

### DISCUSSION THREAD

**Discussion by DragonOsman** [vercel/community#4317] [Help] [answered]
### Summary

I'm working on a task scheduler app for my sister, as she wanted me to build one she could use for her daughter's tasks.  Just a while ago, though, she checked it and told me that all she sees is a blank screen.  It appears on my laptop, though.  She's using an iPhone 11, so I checked its screen size and tried to view it in that size on Dev Tools (that mode where you can preview a page on different screen sizes), and I verified that it at least appears.  

My app's [repository link](https://github.com/DragonOsman/task-scheduler).
Anyone got any ideas? Thanks.

### Example

https://task-scheduler-dragonosman.vercel.app

### Steps to Reproduce

Just look at it on any computer or mobile device.  This is such a weird issue for me that I don't have anything else to say here.

**Comment by DragonOsman** [comment]
Adding for clarification: my sister told me she can view the app just fine on her desktop, so the problem is with iPhone.  I need to know what possible reasons there could be for this to happen.

**Reply by amyegan** [reply, marked as answer]
I just checked and was able to view the site on Safari on an iPhone as well as multiple browsers on my desktop. I see you have made a few more commits since posting your original question, so it looks like you already solved the problem. 🙏

**Reply by maiconcarraro** [reply]
@DragonOsman what was the solution?
